manifests/plugin/abrt/params.pp

```diff
@@ -1,6 +1,7 @@
 # Default parameters for the Pulp smart proxy plugin
 class foreman_proxy::plugin::abrt::params {
   $enabled                 = true
+  $version                 = 'installed'
```

Comment:
I wonder if we should default to ```undef``` so we actually use the default in ```manifests/plugin.pp```. If that default is changed to ```foreman_proxy::plugin_version``` it would allow users to easily switch from installed to latest for every plugin.